### PR TITLE
fix: show full event message in pod event view (#12104)

### PR DESCRIPTION
**Pull request by schakrad**
If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)). 

Screen Shot 2023-02-02 at 6 02 44 PM
Note on DCO:

### CODE REVIEW

File: ui/src/app/applications/components/application-details/application-resource-list.tsx [hunk 1]
```diff
@@ -75,19 +75,18 @@ export const ApplicationResourceList = ({
...
+                                        name: res.name,
+                                        version: res.version,
+                                        kind: res.kind,
+                                        namespace: res.namespace,
+                                        group: res.group,
+                                        info: null,
+                                        uid: '',
+                                        resourceVersion: null,
+                                        parentRefs: []
+                                    })}
```
Comment (ciiay) [inline ui/src/app/applications/components/application-details/application-resource-list.tsx:89]: Is this change intentional?

**Review by ciiay — COMMENTED** [review]
LGTM
Just left a small comment.

File: ui/src/app/applications/components/application-details/application-resource-list.tsx (hunk 1 above)
Comment (schakrad) [reply ui/src/app/applications/components/application-details/application-resource-list.tsx:89]: This change is not related to this PR. This file is in the same state as before. I didn't make any change in the code.

**Comment by alexef** [comment]
This is very useful. I believe we have similar issues (text is truncated with elipsis) in other places, such as:
- Application Conditions dialog
- Sync Status dialog

Can you apply the same fix there, or should we make it a separate code change?

**Comment by schakrad** [comment]
> This is very useful. I believe we have similar issues (text is truncated with elipsis) in other places, such as:
> 
> * Application Conditions dialog
> * Sync Status dialog
> 
> Can you apply the same fix there, or should we make it a separate code change?

@alexef , Could you please raise the above as a new issue?

**Review by ciiay — APPROVED** [review]
LGTM

**Review by crenshaw-dev — APPROVED** [review]
Thanks @schakrad!

**Comment by crenshaw-dev** [comment]
Cherry-picked onto release-2.6 for 2.6.3, release-2.5 for 2.5.12, and release-2.4 for 2.4.24.
